### ISSUE TITLE
Document values available in templates

### ISSUE THREAD

**Issue by frangio** [OpenZeppelin/solidity-docgen#258]

**Comment by paulrberg** [comment]
Yes, please. I just stumbled upon Uniswap's [contract.hbs](https://github.com/Uniswap/docs/blob/400e0d7c52cab461218abc00e40120c9606b7678/contract.hbs) template, and it took me a minute to realize that keywords like `visibility` are custom values used by `solidity-docgen` to generate the Markdown report.
